libnetwork/libnetwork_internal_test.go

```diff
@@ -490,7 +491,7 @@ func TestServiceVIPReuse(t *testing.T) {
 
 	ipToResolve := netutils.ReverseIP("192.168.0.1")
 
-	ipList, _ := n.ResolveName("service_test", types.IPv4)
+	ipList, _ := n.ResolveName(context.Background(), "service_test", types.IPv4)
```

Comment:
do we want these to ultimately be used with OTEL during the tests as well? (just curious; I know we added OTEL to the integration tests, I don't know if we want to do the same for other tests)

Reply:
Can we pre-declare the context with `ctx := context.TODO()`?

Reply:
Ah, thanks for chiming in; had your tweet in mind indeed (and that would be my next question if the answer was "yes, we want to punch it through to these tests") 👍

Reply:
Wondering, what's the advantage of using `context.TODO()` instead of `context.Background()` here? In the API, daemon, integration tests, etc... I think it makes sense but for unit tests, I don't see the point.

Reply:
I think the idea here was that we can have a parent context for the test (which we did for integration tests), but @cpuguy83 may be able to fill me in on that one.

Reply:
Oh, actually don't get me wrong: I'm not against assigning the context to a var. I'll do that change.

Reply:
Done!